### PR TITLE
[FLASK] Create new E2E test for snap_getLocale

### PR DESCRIPTION
**Pull request by bowensanders**
Creates a new E2E test for the snap_getLocale function.

### CODE REVIEW

File: test/e2e/snaps/enums.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   TEST_SNAPS_WEBSITE_URL:
-    'https://metamask.github.io/snaps/test-snaps/0.38.0-flask.1/',
+    'https://metamask.github.io/snaps/test-snaps/0.39.0-flask.1/',
```
Comment (FrederikBolding) [inline test/e2e/snaps/enums.js:3]:
```suggestion
    'https://metamask.github.io/snaps/test-snaps/0.39.1-flask.1/',
```
Try this 😊

File: test/e2e/snaps/test-snap-get-locale.spec.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        );
+        await driver.delay(500);
+
+        // check dialog contents
+        const result = await driver.findElement('.snap-ui-renderer__panel');
+        await driver.scrollToElement(result);
+        await driver.delay(500);
+        assert.equal(
+          await result.getText(),
+          'Hello https://metamask.github.io!\nThis is a dialog!',
```
Comment (FrederikBolding) [inline test/e2e/snaps/test-snap-get-locale.spec.js:98]: Could we switch the language to something else and verify that it also works?

**Review by FrederikBolding — DISMISSED** [review]

**Review by FrederikBolding — CHANGES_REQUESTED** [review]
Wait with merging this until https://github.com/MetaMask/metamask-extension/pull/21019/ is merged

**Review by FrederikBolding — DISMISSED** [review]

File: test/e2e/snaps/enums.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   TEST_SNAPS_WEBSITE_URL:
-    'https://metamask.github.io/snaps/test-snaps/0.38.0-flask.1/',
+    'https://metamask.github.io/snaps/test-snaps/0.39.1-flask.1/',
```
Comment (Mrtenz) [inline test/e2e/snaps/enums.js:3]:
```suggestion
    'https://metamask.github.io/snaps/test-snaps/1.0.0/',
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: test/e2e/snaps/enums.js (hunk 3 above)
Comment (FrederikBolding) [reply test/e2e/snaps/enums.js:3]: Oh I must have forgotten to change this in my PR 😅

**Review by Mrtenz — DISMISSED** [review]

**Review by FrederikBolding — APPROVED** [review]

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**